ci.sh

```diff
@@ -11,14 +11,14 @@ main() {
     apt install -y python3-autopep8
   fi
 
-  ./podman-build.sh
```

Comment:
The thing is about this is, it doesn't build ramalama, it only builds container images...

Maybe container-build.sh ?

Not a blocking issue for this PR anyway

Reply:
We can catch in a follow on PR if needs be